packages/workspace/src/tasks-runner/task-orchestrator.ts

```diff
@@ -314,6 +314,11 @@ export class TaskOrchestrator {
           task: result.task,
           status: result.status,
           code,
+          terminalOutput:
```

Comment:
we have a separate lifecycle event called printTaskTerminalOutput, which is currently invoked. Can we use that one instead? 

For instance, if we forward output, we cannot print it. 

```
if (!forwardOutput) {
this.options.lifeCycle.printTaskTerminalOutput(
task,
TaskCacheStatus.NoCache,
terminalOutput
);
}
```

We talked about that the Ink terminal output renderer should not be used with tasks that are forwarded (at least in the case of run many), but even with that, let's use the other life cycle event.